### DISCUSSION TITLE
Create an order without having to add items into order

### DISCUSSION THREAD

**Discussion by damiannineseven** [vendurehq/vendure#1289] [Q&A] [answered]
Hi Everyone, We are trying to use Vendure as a Back office solution.
My storefront is calling products from a different website.
I want to create order from my Storefront and send it to vendure admin WITHOUT creating a product in the order. I would like to just paste my product info to the description or something like that in the order. Is this possible?

**Comment by damiannineseven** [comment, marked as answer]
The question was answered by Michael Bromley on the Vendure slack support channel.
I will quote his answer here in case anyone is wondering the same thing

> This is not really possible, because in Vendure an Order contains OrderLines and each OrderLine must point to a particular ProductVariant, which in turn must be a member of a Product. As you can see, you can't have just one part without also needing the entire dependency graph.
But what you could do is to create products on-the-fly as new orders come in. You'd of course end up with lots of products, since they would just accrue indefinitely.

**Comment by u2ptester** [comment]
Hi guys, first of all thanks alot for the great job and delivering such a good solution!

I think, my question belongs to this one, that's why I post it here.

I assume then a product variant can not be removed when it is already referenced by an order line, right?
And what happens to the old orders if the admin user changes the price, product name and description at a later time point, for example?

I wanted to try it on my own but unfortunately the demo environment throws following error message there:
_The provided credentials are invalid_invalid_

Kind regards

**Reply by michaelbromley** [reply]
Hi, thank you for the kind words :)

First of all, the demo should be working now, I just reset it.

And to answer your questions: Yes, products cannot be removed. Deleting a product is a "soft delete" - the actual record remains in the DB to ensure referential integrity. Price changes do not matter, since the price is recorded at the moment of adding to the Order as OrderLine.price. Changes to the name of the variant _will_ affect display of historical orders, however. If this is an issue, a work-around would be to add a custom field to the OrderLine entity and record the variant name during checkout.

**Reply by u2ptester** [reply]
Hi Michael, thanks a lot for replying so quick :)

**Reply by u2ptester** [reply]
I think this can be a problem when we implement, e.g. an order history and order details page for the customer section/portal. Anyway, the work-aroud you propose will help to fix it :)
